debian/linglong-bin.postinst

```diff
@@ -1,98 +1,101 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
https://stackoverflow.com/questions/34193030/should-deb-package-maintainer-script-use-bash-or-sh

> In conclusion, I think you shouldn't bother about your scripts being compatible with tcsh and the best practice would be to only use /bin/sh for all your packaging needs.